bnd.bnd

```diff
@@ -0,0 +1,10 @@
+Bundle-SymbolicName: org.jspecify.jspecify
+Bundle-Name: JSpecify annotations
+Bundle-Vendor: jspecify.dev
+Bundle-Description: An artifact of well-specified annotations to power static analysis checks and JVM language interop.
+Bundle-DocURL: https://jspecify.dev/docs/start-here
```

Comment:
```suggestion
Bundle-DocURL: https://jspecify.org/docs/start-here
```

Reply:
https://jspecify.org doesn't work :( The redirect is active only at the plain-http http://jspecify.org. If we want https, we need to point straight to the .dev site.

Reply:
Ah :-(
#267 would fix that.

Reply:
So I leave it as "Bundle-DocURL: https://jspecify.dev/docs/start-here" for now? (until the .org tld is working in the future)

Reply:
> https://jspecify.org doesn't work :( The redirect is active only at the plain-http http://jspecify.org. If we want https, we need to point straight to the .dev site.

I leave it as is. It can be changed at any time in the future without causing a problem.